identity/ideth/service.go

```diff
@@ -336,7 +337,7 @@ func (i service) Exists(ctx context.Context, did id.DID) error {
 }
 
 // ValidateKey checks if a given key is valid for the given centrifugeID.
-func (i service) ValidateKey(ctx context.Context, did id.DID, key []byte, purpose *big.Int) error {
+func (i service) ValidateKey(ctx context.Context, did id.DID, key []byte, purpose *big.Int, validateAt *time.Time) error {
```

Comment:
`validateAt` -> `validAt` seems less confusing to me

Reply:
its `validateAt`, since we are saying validate at a particular time

Reply:
we can also make it just `at`